test/README.md

```diff
@@ -1,30 +1,64 @@
-# Testing
+# Polkadot Conformance Testsuite
 
-The `test/` directory contains tests for different components from different implementations such as Rust, C++ and Golang:
+This `test/` directory contains specification tests of the different components of Polkadot, which are run against the different implementations (i.e in Rust, C++ and Golang) of Polkadot
 
-- SCALE Codec
-- State Trie
-- Polkadot Host API
+Currently the testsuite contains the following tests:
 
-This ensures that the different implementations behave the same and produce the identical output.
+- SCALE Codec Encoding (scale-codec)
+- State Trie Hashing (stat-trie)
+- Polkadot Host API (hostapi and hostapi-legacy)
 
-## Local container (CircleCI)
+This ensures that the different implementations behave in the same way and produce the identical output. 
 
-The easiest way to run those tests is by using `docker` via the [local-ci tool](https://circleci.com/docs/2.0/local-cli/). No API tokens are required for running the suite.
+# Dependencies
+
+The test suite depends on the following components:
+
+- _julia_ to run the testsuite
+- _rust-nightly_ (with wasm target) to build substrate 
+- _cmake_ to build kagome
+- _go_ to build gossamer
+
+While the official target of our testsuite are currently only debian-based systems, there is in general no reason for it to not be able to run on  any recent GNU/Linux or even UNIX-based OS, like OS X.
+
+## General Build
+
+Each of the API adapters has to be build (see [adapters subfolder](./adapters/)), before the testsuite can be run.
```

Comment:
I'm not sure what is the advantage of not havig the option to build and run all tests locally.

Reply:
You always had that option, but I agree it was a bit convoluted having to go to each of the adapter subfolders to build the adapters. Now fixed with a Makefile.